### DISCUSSION TITLE
Compile Vue?

### DISCUSSION THREAD

**Discussion by HeadStudios** [whitecube/nova-flexible-content#396] [General]
Random question if I wanted to compile Vue would that be possible like if I wanted to make an edit or something? Thanks!

**Comment by voidgraphics** [comment]
Not sure what you mean, could you elaborate? What are you trying to achieve?

**Reply by HeadStudios** [reply]
Hey never mind I forked it on Github then pulled it and did composer install with my Nova password then pulled it and now compiling vue but then I just copy paste compiled field.js file or whatever into my vendor folder on my site. Btw dependsOn from drop down at the moment I'm doing through Vue - would be cool to have templates that I could pass and have the flexible repeater fields create rows automatically based on drop down selected. Awesome plugin by the way!
